thavalon-webapp/src/components/Navbar.tsx

```diff
@@ -1,9 +1,13 @@
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import React, {useState} from 'react';
+import React from 'react';
 import {NavLink} from 'react-router-dom';
 import styled from 'styled-components';
 import {faBars} from '@fortawesome/free-solid-svg-icons';
 
+/**
+ * Code for supporting hamburger menu on mobile.
+ * @param event The event caused by clicking button.
+ */
 function openMenu(event: any) {
     let x = document.getElementById("navbar");

```

Comment:
I think there's a way to do this more in a more Reacty manner. Check out refs for accessing DOM elements directly. Might work here. Also we can figure out the type for event to type it.

X also should be constant as it's never mutated.

https://www.newline.co/@dmitryrogozhny/how-to-access-dom-nodes-in-react-with-refs--57d97b3d

Reply:
If we're only changing CSS classes, we could also make that state that's updated here and avoid touching DOM nodes at all.